crates/ruff_python_formatter/src/other/arguments.rs

```diff
@@ -103,7 +101,12 @@ impl FormatNodeRule<Arguments> for FormatArguments {
                 // )
                 // ```
                 // TODO(konstin): Doesn't work see wrongly formatted test
```

Comment:
@konstin - Is this still applicable? I can't tell what it applies to.

Reply:
i don't think think so